doc/nrf/app_boards.rst

```diff
@@ -54,6 +54,8 @@ Also see the :ref:`zephyr:boards` section in the Zephyr documentation.
 |                   |            |                                                                 |                                       |
 |                   |            |                                                                 | ``nrf5340dk_nrf5340_cpuapp_ns``       |
 +-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| Thingy:53         | PCA20053   | :ref:`thingy53_nrf5340 <zephyr:thingy52_nrf52832>`              | ``thingy53_nrf5340``                  |
```

Comment:
Shouldn't we align with the nRF5340 DK and list all options like `thingy53_nrf5340_cpuapp`, `thingy53_nrf5340_cpuapp_ns` and `thingy53_nrf5340_cpunet`?

Reply:
Yes, there's also the wrong link there. Let me fix.